src/amo/pages/Guides/index.js

```diff
@@ -116,23 +128,35 @@ export class GuidesBase extends React.Component<InternalProps> {
   getGuidesSections = (
     sections: Array<SectionsType>,
   ): React.ChildrenArray<React.Node> => {
-    const { clientApp, i18n, lang } = this.props;
+    const { addons, clientApp, i18n, lang } = this.props;
 
     return sections.map((section) => {
       // TODO: look into having these links use the Router (vs 'a' tag).
       // See https://github.com/mozilla/addons-frontend/issues/6787.
-      const exploreMoreLink = i18n.sprintf(section.exploreMore, {
-        linkStart: `<a class="Guides-section-explore-more-link" href="/${lang}/${clientApp}${
-          section.exploreUrl
-        }">`,
-        linkEnd: '</a>',
-      });
+      let exploreMoreLink;
+      if (lang && clientApp) {
+        exploreMoreLink = i18n.sprintf(section.exploreMore, {
+          linkStart: `<a class="Guides-section-explore-more-link" href="/${lang}/${clientApp}${
+            section.exploreUrl
+          }">`,
+          linkEnd: '</a>',
+        });
+      }
+
+      const addon = addons[section.addonGuid];
 
       return (
         <div className="Guides-section" key={section.exploreUrl}>
           <h2 className="Guides-section-title">{section.header}</h2>
           <p className="Guides-section-description">{section.description}</p>
 
+          {addon && (
```

Comment:
I had to add this check to make `AMInstallButton` happy. My concern is for when we add in LoadingText...

Reply:
okay I was to work around this by assigning a default to the addon var in `GuidesAddonCard` along with returning `addon` in its mapStateToProps (see https://github.com/mozilla/addons-frontend/pull/6859#discussion_r233868705). I think this is the better way to go for when we are dealing with this LoadingText